wps-cloudformation-template.yaml

```diff
@@ -384,12 +384,20 @@ Resources:
   JobQueue:
     Type: AWS::Batch::JobQueue
     Properties:
-      JobQueueName: !Sub 'JavaDuckQueue1-${AWS::StackName}'
+      JobQueueName: !Sub 'JavaDuckExecuteQueue-${AWS::StackName}'
       Priority: 2
       ComputeEnvironmentOrder:
         - Order: 1
-          ComputeEnvironment: !Ref ComputeEnvironment
-  ComputeEnvironment:
+          ComputeEnvironment: !Ref JobComputeEnvironment
+  TestQueue:
+      Type: AWS::Batch::JobQueue
+      Properties:
+        JobQueueName: !Sub 'JavaDuckTestQueue-${AWS::StackName}'
+        Priority: 2
+        ComputeEnvironmentOrder:
+          - Order: 1
+            ComputeEnvironment: !Ref TestComputeEnvironment
+  JobComputeEnvironment:
```

Comment:
May be good to add a comment suggesting updating the TestComputeEnvironment as well and vice versa.  https://aws.amazon.com/blogs/devops/adding-comments-inside-aws-cloudformation-templates/